app/translations/index.ts

```diff
@@ -82,3 +83,25 @@ function deepEncode (obj: any): any {
 export function encodeScope (text: string): string {
   return Buffer.from(text).toString('base64')
 }
+
+export function getLocaleByLanguageName (languageName: EnvironmentLanguage): string {
+  switch (languageName) {
```

Comment:
Would it make sense to use the language key instead of language name as value of the list? Some countries will have same locale key